test/shopify-cli/commands/config_test.rb

```diff
@@ -62,6 +62,18 @@ def test_will_disable_analytics_that_is_enabled
         run_cmd("config analytics --disable")
         refute ShopifyCli::Config.get_bool('analytics', 'enabled')
       end
+
+      def test_will_enable_shopifolk_beta_that_is_disabled
+        ShopifyCli::Config.set('shopifolk-beta', 'enabled', false)
```

Comment:
Should we be setting this up to be `disabled` at first?

Reply:
We don't write 'disabled' to the config, it's just `enabled = false`.

Reply:
I realised some of these tests were invalid, they were writing the config wrong. Fixed in 99654c15.

Reply:
No that isn't right either, going to revert...

Reply:
Ok, so when we run `shopify config shopifolk-beta --enable` it writes:

```
[shopifolk-beta]
  enabled = true
```

and when we run `shopify config shopifolk-beta --disable` it writes

```
[shopifolk-beta]
  enabled = true
```

So we never actually write `disabled` to the config.

Reply:
My mistake there, I missed the booleans when reading this.